package.json

```diff
@@ -24,11 +24,5 @@
     "changesets-changelog-clean": "^1.3.0",
     "typescript": "^5.8.2"
   },
-  "packageManager": "pnpm@10.4.0",
-  "pnpm": {
-    "onlyBuiltDependencies": [
-      "@biomejs/biome",
-      "esbuild"
-    ]
-  }
```

Comment:
these options can now live in pnpm-workspace in case you're wondering where they went 😅